README.md

```diff
@@ -9,7 +9,43 @@ A multi-language library containing implementations of common software design an
 
 ## Articles / Websites
 
+### Model View View-Model (MVVM) - Architectural Pattern
 
+Just like the Model View Controller (MVC) pattern, Model View View-Model (MVVM) is an architectural pattern in software development whose purpose is to provide a clear separation of concerns in applications, especially those with complex user interfaces and interactive features. In both patterns, the program logic of your application (the model) is kept separate from the user interface (the view). The View-Model (and Controller in the case of the MVC pattern) act as the mediator between the Model and View. This separation enhances the structure and maintainability of the code, making it easier to develop, test and manage.  
+
+**Key purposes and benefits:**
+* Reduces dependencies and conflicts 
+* Facilitates testability
+* Improves maintainability
+* Data binding reduces need for boilerplate code
+* Enhanced UI responsiveness
+* Simplifies UI state management
+* Facilitates rich and interactive UIs
+
+#### Key Differences and Improvements on MVC pattern
+
+The MVVM pattern differs from MVC, however, in two key respects. First, in the MVVM pattern, the view is completely decoupled from the model and relies heavily on data binding to keep the View and ViewModel synchronized. Changes in the ViewModel automatically reflect in the View through data binding without needing explicit instruction. This is a key benefit to MVVM and enables it to create dynamic responsive UIs. 
```

Comment:
I feel like using  `, however,` is a redundancy that makes reading a little harder.